src/map/status.cpp

```diff
@@ -10659,7 +10655,7 @@ int32 status_change_start(struct block_list* src, struct block_list* bl,enum sc_
 			unit_data *ud = unit_bl2ud(bl);
 
 			if (ud)
-				ud->attackabletime = ud->canact_tick = ud->canmove_tick = gettick() + tick;
+				ud->attackabletime = ud->canact_tick = ud->canmove_tick = ud->endure_tick = gettick() + tick;
```

Comment:
```suggestion
				ud->attackabletime = ud->canact_tick = ud->canmove_tick = gettick() + tick;
				ud->endure_tick = gettick() + tick;
```

Reply:
Why?

Reply:
Track changes / git blame.
If you want something like this, please define a variable above and asign it to all.